src/routes/personal/personal-phone-numbers-check-routes.js

```diff
@@ -0,0 +1,31 @@
+import { personalPhoneNumbersCheckPresenter } from '../../presenters/personal/personal-phone-numbers-check-presenter.js'
+import { fetchPersonalChangeService } from '../../services/personal/fetch-personal-change-service.js'
+import { updatePersonalPhoneNumbersChangeService } from '../../services/personal/update-personal-phone-numbers-change-service.js'
+
+const getPersonalPhoneNumbersCheck = {
+  method: 'GET',
+  path: '/account-phone-numbers-check',
```

Comment:
This one confuses me. I can see on the design spec that it says to call the route 
`account-phone-numbers-check`, but I can't understand why it isn't consistent with the rest of the routes, maybe one to just ask on?